dashboard/app/models/pd/workshop.rb

```diff
@@ -679,6 +679,10 @@ def local_summer?
     subject == SUBJECT_SUMMER_WORKSHOP
   end
 
+  def csp_wfrt?
```

Comment:
Can we have a comment defining this acronym?

Reply:
will do